string robot_name
string robot_link_name
string object_name
---
bool ok
